### PR TITLE
Add chromatic story with all icons

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 
In preparation to being able to upgrade icon packages again in the future. Since we'll always pin our version now, we can rely on chromatic tests so we know if any particular update breaks anything.

It's broken up alphabetically with large heights so that if a single icon is added/removed we'll know without it affected every single other icon in the chromatic. Instead, it'll only affect icons after it in it's section of the alphabet.

Also contains a dialog style fix that was broken in chromatic

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Review by reidbarber — APPROVED** [review]
LGTM. Chromatic build looks good, should be really useful

File: packages/@spectrum-icons/build-tools/generateIcons.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function generateIcons(iconDir, outputDir, nameRegex, template) {
     // generate index barrel
     let indexFile = iconFiles.map(icon => {
       let iconName = icon.substring(0, icon.length - 3);
-      return `export * from './${iconName}';\n`;
+      return `export * as ${isNaN(Number(iconName[0])) ? iconName : `_${iconName}`} from './${iconName}';\n`;
```
Comment (LFDanLu) [inline packages/@spectrum-icons/build-tools/generateIcons.js:50]: just to confirm, this change here is so you can do iterate through all the icons in the icons/workflow package like you've done in the chromatic story right? And the NaN check is for icons like `123`

**Review by LFDanLu — COMMENTED** [review]
Just a clarifying question, but otherwise looks good to me

File: packages/@spectrum-icons/build-tools/generateIcons.js (hunk 1 above)
Comment (snowystinger) [reply packages/@spectrum-icons/build-tools/generateIcons.js:50]: correct, their actual name is A4u123, but we remove the A4u prefix, and `123` isn't a valid function name, so I have to prefix it with the `_`

**Review by LFDanLu — APPROVED** [review]

**Pull request merged**